dts/bindings/gpio/gpio-controller.yaml

```diff
@@ -12,3 +12,7 @@ properties:
       type: int
       required: true
       description: Number of items to expect in a GPIO specifier
+    ngpios:
```

Comment:
@galak Seems like this could usefully be `default: 32` so that we don't have to touch the whole world to support the status quo.

Reply:
I'm ok if we want to do this as part of #20115 (once topic-gpio is rebased).  However, I think its a short term solution until we get SoC DTSs updated properly.